docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+
+2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is outdated by a newer ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating a one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
+
+3. One or more ALZ custom policies, assigned via ALZ custom policy initiative, are outdated by newer ALZ custom policy(s) with **updated parameters**. The process for managing this is described in [Updating ALZ custom policies in ALZ custom policy initiative to newer ALZ custom policies](need a reference)
+
+### Updating a one or more ALZ custom policies to newer ALZ custom policy
+
+For this scenario we will use the ALZ custom policy *Deploy Diagnostic Settings for WVD Host Pools to Log Analytics workspace*.
+
+Considering no parameters have changed, this is a simple exercise that consists of replacing the policy definition content with the latest policy definition. While it is possible to update the policy via the portal GUI, there will be a lot of copying and pasting content. To minimize errors, we will be updating this policy via a PowerShell script.
+
+Before we begin, we need to identify the policy definition name and location to be used in our PowerShell script below.
+
+- Go to [Azure Portal](https://portal.azure.com)
+- Open Policy
+- Go to Definitions and in Search, find the ALZ custom policy.
+
+  ![alz-custom-policy-def-search](media/1.1.update-alz-custom-policy-def-search.png)
+
+- Click on the hyperlink for the policy definition
+- Capture the policy definition name and location from `Definition ID` and `Definition location`. In this example, the `Definition ID` is `/providers/Microsoft.Management/managementGroups/MTB/providers/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-WVDHostPools` with a policy definition name of **Deploy-Diagnostics-WVDHostPools** and a location of **MTB**. The policy definition name is the set of characters following the last `/`. Both the policy definition name and location will be used in the PowerShell script below.
+
+  ![alz-custom-policy-def-name](media/1.2.update-alz-custom-policy-def-name.png)
+
+- To update to the latest version of the definition, we will use the policy definition templates available in https://github.com/Azure/Enterprise-Scale/tree/main/src/resources/Microsoft.Authorization/policyDefinitions as described in the following.
+
+- Go to https://portal.azure.com
+- Start an Azure Cloud Shell with PowerShell engine
+- Execute the following PowerShell script for each ALZ custom policy definition:
```

Comment:
I think we need to make sure there is a link to a disclaimer here regarding support and running these scripts

Reply:
https://github.com/Azure/Enterprise-Scale/blob/main/SUPPORT.md